docs/source/changelog.rst

```diff
@@ -604,7 +604,7 @@ Version 0.6.0
   file reads in blocks (:pr:`191`).
 * Fixed equality checks for file system instance to return ``False`` when compared to objects
   other than file systems (:pr:`192`)
-* Fixed a bug in :meth:`fsspec.FSMap.keys` returning a generator, which was consumed upon iteration (:pr:`189`).
+* Fixed a bug in ``fsspec.FSMap.keys`` returning a generator, which was consumed upon iteration (:pr:`189`).
```

Comment:
Function no longer exists as it is inherited from `collections.abc.MutableMapping`.